waspc/data/Generator/templates/react-app/src/actions/index.ts

```diff
@@ -21,7 +21,7 @@ export type ActionOptions<ActionInput, CachedData> = {
 /**
  * A documented (public) way to define optimistic updates.
  */
-export type OptimisticUpdateDefinition<ActionInput, CachedData = unknown> = {
+export type OptimisticUpdateDefinition<ActionInput, CachedData> = {
```

Comment:
This file contains unrelated fixes for a bug that came to light when I introduced the entities to the frontend.